### DISCUSSION TITLE
support for multisig transactions on dogecoin?

### DISCUSSION THREAD

**Discussion by majinbruce** [dogecoin/dogecoin#3298] [Help needed]
after setting up a dogecoin node is there support for descriptors?? or even normal multisig transaction signings?

**Comment by iamtiagogalvao** [comment]
Dogecoin, does support multisignature (multisig) transactions, which require multiple private keys to authorize a transaction.

**Reply by majinbruce** [reply]
I could not find any resources to perform multisg transaction, I've done the node set up with descriptors enabled, can you share any resources on how to perform multisg

**Reply by iamtiagogalvao** [reply]
Have you read #336 ? It is quite old, but maybe you can find useful information there.
